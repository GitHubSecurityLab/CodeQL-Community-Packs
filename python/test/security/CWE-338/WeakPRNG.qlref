security/CWE-338/WeakPRNG.ql