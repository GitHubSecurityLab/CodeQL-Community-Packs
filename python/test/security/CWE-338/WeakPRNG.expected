| app.py:6:1:6:16 | ControlFlowNode for Attribute() | Using weak PRNG |
| app.py:11:1:11:15 | ControlFlowNode for Attribute() | Using weak PRNG |
| app.py:12:1:12:23 | ControlFlowNode for Attribute() | Using weak PRNG |
| app.py:13:1:13:21 | ControlFlowNode for Attribute() | Using weak PRNG |
| app.py:15:1:15:20 | ControlFlowNode for Attribute() | Using weak PRNG |
| app.py:18:1:18:12 | ControlFlowNode for Attribute() | Using weak PRNG |
| app.py:19:1:19:44 | ControlFlowNode for Attribute() | Using weak PRNG |
